src/platform/context/egl.c

```diff
@@ -176,8 +176,8 @@ create_context(struct wlc_backend_surface *bsurface)
             EGL_RED_SIZE, 1,
             EGL_GREEN_SIZE, 1,
             EGL_BLUE_SIZE, 1,
-            EGL_ALPHA_SIZE, 0,
-            EGL_DEPTH_SIZE, 0,
+            EGL_ALPHA_SIZE, 1,
```

Comment:
You should not need this for anything, unless you plan to render the compositor as transparent window over X11 or other compositor. This simply means that the color buffer for context will have alpha component, which does not make sense when rendering to TTY without anything behind.